app/models/mhv_account.rb

```diff
@@ -128,7 +128,8 @@ def create_mhv_account!
       if client_response[:api_completion_status] == 'Successful'
         StatsD.increment("#{STATSD_ACCOUNT_CREATION_KEY}.success")
         user.va_profile.mhv_ids = [client_response[:correlation_id].to_s]
-        user.instance_variable_get(:@mvi).save
+        mvi_response = user.instance_variable_get(:@mvi).instance_variable_get(:@mvi_response)
```

Comment:
Since User has access to an instance of Mvi how about we make Mvi#mvi_response public and then add a method on user that re-caches? (should be able to avoid the `instance_variable_get`s then)